### PR TITLE
Add policy on successful validation

### PR DESCRIPTION
**Pull request by jaceks2106**
Changes:
* Bump version to 0.3
* Custom and RateLimit policies are created for Products and APIs
* Sample publish.yml file is updated
* Updated the latest change with *entity*Id to general `EntityId` property in validation result model

### CODE REVIEW

File: src/Kmd.Logic.Gateway.Automation/Publish.cs [hunk 1]
```diff
@@ -102,22 +102,84 @@ private async Task CreateOrUpdateProducts(Guid subscriptionId, Guid providerId,
...
+                    }
+                }
+
+                if (customPolicies != null)
+                {
+                    foreach (var customPolicy in customPolicies)
+                    {
+                        var customPolicyResult = policiesResults.CustomPolicies.Single(cp => string.Equals(cp.Name, customPolicy.Name, StringComparison.OrdinalIgnoreCase));
+
+                        using var xmlFs = new FileStream(path: Path.Combine(folderPath, customPolicy.XmlFile), FileMode.Open, FileAccess.Read);
```
Comment (rshaiju) [inline src/Kmd.Logic.Gateway.Automation/Publish.cs:160]: A subtle improvement chance. StreamReader has an overload that accepts the file path. Stream is not needed in that case

File: src/Kmd.Logic.Gateway.Automation/Publish.cs [hunk 2]
```diff
@@ -102,22 +102,84 @@ private async Task CreateOrUpdateProducts(Guid subscriptionId, Guid providerId,
             foreach (var productValidationResult in productValidationResults)
```
Comment (rshaiju) [inline src/Kmd.Logic.Gateway.Automation/Publish.cs:102]: I feel this class is growing beyond what it should. Probably this is the right time to refactor the responsibility into smaller ones that take care of products, apis and so on.. Not a review comment as such. But just a thought!!

Comment (jaceks2106) [reply src/Kmd.Logic.Gateway.Automation/Publish.cs:102]: You are definitely right. We can make a task for that.

File: src/Kmd.Logic.Gateway.Automation/Publish.cs (hunk 1 above)
Comment (jaceks2106) [reply src/Kmd.Logic.Gateway.Automation/Publish.cs]: <3 Done.

File: src/Kmd.Logic.Gateway.Automation/Publish.cs (hunk 2 above)
Comment (jaceks2106) [reply src/Kmd.Logic.Gateway.Automation/Publish.cs:102]: Created.

**Review by rshaiju — APPROVED** [review]

**Pull request merged**